README.md

```diff
@@ -62,6 +62,10 @@
 
 - [Live Demo Link](https://yourdeployedapplicationlink.com)
 
+## 🌐 Online version
+
+- [Ezra-Portfolio](https://ezraermy.github.io/)
+
```

Comment:
- [Optional] Great Job for adding your live version here, however, it is better to use the already prepared link line `63` and replace your link instead of creating the new section to follow [professional readme](https://github.com/microverseinc/readme-template).👌👍